ts/session/profile_manager/ProfileManager.ts

```diff
@@ -115,9 +115,7 @@ async function updateOurProfileDisplayNameOnboarding(newName: string) {
     const appliedName = await UserConfigWrapperActions.getName();
 
     if (isNil(appliedName)) {
-      throw new Error(
-        'updateOurProfileDisplayNameOnboarding failed to retrieve name after setting it'
-      );
+      throw new Error('failed to retrieve display name after setting it');
```

Comment:
you should create a custom error in  ts/session/utils/errors.ts, and do an instanceof instead.

Reply:
Fixed